endpoints-support.md

```diff
@@ -475,6 +494,35 @@
 |<sub>/rest/storage-volume-templates/{id}</sub>                                           | PUT      | :white_check_mark: | :white_check_mark: |    |
 |<sub>/rest/storage-volume-templates/{id}</sub>                                           | DELETE   | :white_check_mark: | :white_check_mark: |    |
 |<sub>/rest/storage-volume-templates/{id}/compatible-systems</sub>                        | GET      | :heavy_minus_sign: | :heavy_minus_sign: |    |
+|     **Switch Types**                                                                                                                              |
+|<sub>/rest/switch-types</sub>                                                            | GET      | :white_check_mark: | :white_check_mark: |    |
+|<sub>/rest/switch-types/{id}</sub>                                                       | GET      | :white_check_mark: | :white_check_mark: |    |
+|     **Switches**                                                                                                                                   |
+|<sub>/rest/switches</sub>                                                                | GET      | :white_check_mark: | :white_check_mark: |    |
+|<sub>/rest/switches/{id}</sub>                                                           | GET      | :white_check_mark: | :white_check_mark: |    |
+|<sub>/rest/switches/{id}</sub>                                                           | PATCH    | :heavy_minus_sign: | :black_square_button: |    |
```

Comment:
Actually this endpoint it is supported in Java, even though no method was added to the sample.

Reply:
Updated